Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1

```diff
@@ -197,15 +197,13 @@ param (
 
     try
     {
-        if($isFileCopy -and $doCleanUp -and (Test-Path -path $destinationNetworkPath -pathtype container))
+        if($doCleanUp -and (Test-Path -path $destinationNetworkPath -pathtype container))
```

Comment:
Updates to L0?

Reply:
This is job we have already mocked job L0.